internal/project/testdata/git-project/.git.bkp/objects/82/0729cec4e8387ac3462762fe726cb8297997c8

```diff
@@ -0,0 +1 @@
+x��Qj1D��S述Ȋ�� ��7�-;Y����ܿ.������m߷��i�R�%�#���ѳ�D�nEĭj�d�T���^�8��gr.̩e�Ps���L>eҚ�y�{��÷��Y�_�j9�۳�>
�c�m��m�sd�@�hf;EGy���#���r��|k��_YxL�
```

Comment:
@adambabik what's all this?

Reply:
I wanted to have a real git repository to do integration tests. As it's possible to embed one git repository in another, except for git submodule but then the submodule must be a standalone repo, I just changed `.git` to `.git.bkp` and added it directly. Before running the tests, I do `cp -r .git.bkp .git`.

There are a few nicer alternatives:
- Have it as an actual standalone repo and use submodules.
- Compress and commit a single tar file.
- Create a repo from scratch as a test setup step in the temp dir.

I think the last one should be doable purely using go-git. I will try to do that.

Reply:
All good. If it works, it works. Mainly wanted to make sure it's intentional.

Reply:
Ok, I will fix it in another PR.